### PR TITLE
[FIX] gui: Add a push button class adapted for variable width text

### PR DESCRIPTION
**Pull request by ales-erjavec**
Use it to fix `auto_commit` generated layout.

### CODE REVIEW

File: Orange/widgets/utils/buttons.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from PyQt4.QtGui import QPushButton, QStyle, QStyleOptionButton
+from PyQt4.QtCore import Qt, QSize
+
+
+class VariableTextPushButton(QPushButton):
+    """
+    QPushButton subclass with an sizeHint method to better support settable
+    variable width button text.
+
+    Use this of class instead of the QPushButton when the button will
```
Comment (lanzagar) [inline Orange/widgets/utils/buttons.py]: Use this _-of_ class

**Comment by lanzagar** [comment]
"fix auto_commit generated layout" - what is the best way to see the difference? / where were previous problems noticeable?

**Comment by ales-erjavec** [comment]
> what is the best way to see the difference? / where were previous problems noticeable?

When the  auto commit checkbox is toggled the layout would change. Most noticeable on widgets with fixed size constraint like most learners.

**Comment by lanzagar** [comment]
OK, apparently that was an issue on OSX (looks annoying).
Was already fine on my linux machine, and it remains the same after this fix.
So, :+1: , merging.

**Pull request merged**